webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -81,4 +81,57 @@
             line-height: 24px;
         }
     }
+
+    .ldap-expandable-section {
+        margin: 20px 0;
+
+        .ldap-expandable-section-header {
+            display: flex;
+            align-items: center;
+            gap: 8px;
+        }
+
+        .ldap-expandable-section-toggle {
+            padding: 4px 0;
+            border: none;
+            background: none;
+            color: var(--button-bg);
+            font-weight: 600;
+
+            &:hover {
+                text-decoration: underline;
+            }
+
+        }
+
+        .ldap-expandable-arrow {
+            color: var(--button-bg);
+            transition: transform 0.2s ease;
+            
+            &.open {
+                transform: rotate(90deg);
+            }
+        }
+
+        .ldap-expandable-section-content {
```

Comment:
Note that this isn't accessible because, even if we set all these, the collapsed content is still accessible to keyboard users. It needs to have `display: none` set to prevent that.

If we can't use `details` and `summary` here, you could solve that by making it so the collapsed state has `display: none` and it changes to this as soon as the element starts to expand. I don't know if we have a good example of this in the code yet, but ideally, we'd have a reusable component for it

Reply:
With the new a11y changes to `LDAPExpandableSetting` is this still a major concern? To add `display: none` to this while keeping animations would add complexity (and some js), so I'd like to avoid it if possible.  Let me know!